src/entities/auth/service/auth.service.ts

```diff
@@ -24,4 +24,20 @@ export class AuthService {
       access_token: await this.jwtService.signAsync(payload)
     };
   }
+
+  async signIn(email: string, password: string): Promise<{access_token: string}> {
+    const user = await this.userService.findOne(email);
+    if (!user) {
+      throw new HttpException(statusMessages.USER_NOT_FOUND, HttpStatus.NOT_FOUND);
+    }
+    const isValidPass = await this.userService.checkPassword(password, user.password);
+    if (!isValidPass) {
+      throw new HttpException(statusMessages.INCORRECT_PASSWORD, HttpStatus.BAD_REQUEST);
+    }
+    const payload = {sub: user._id};
```

Comment:
Why `sub`?

Reply:
It is according to JWT standards: "In the JSON Web Token (JWT) standard, the "sub" (subject) claim is a string that identifies the principal that is the subject of the JWT.  The "sub" claim is used to provide some basic information about the context in which the JWT was issued."

Reply:
Okay, let`s skip it for now. Thanks.